### ISSUE TITLE
Error in beforeCreate hook: "TypeError: Cannot read property 'runtime' of undefined"

### ISSUE THREAD

**Issue by ademilter** [f/vue-wait#35]
```js
const vueLoading = new VueLoading({
  useVuex: true,
  vuexModuleName: 'VL',
  registerComponent: false,
  registerDirective: false,
  accessorName: '$vl'
})

new Vue({
  router,
  store,
  i18n,
  vueLoading,
  render: h => h(App)
}).$mount('#app')
```
![screen shot 2018-06-07 at 17 19 53](https://user-images.githubusercontent.com/187922/41105770-13c805c6-6a77-11e8-9fcd-eb19d2bb97ff.png)

**Comment by f** [comment]
If you want to continue using `vuex-loading` (deprecated package) this bug is fixed on `1.0.4`

**Comment by ademilter** [comment]
fixed ✅
